src/main/java/net/mcreator/ui/modgui/ItemGUI.java

```diff
@@ -325,6 +338,10 @@ public ItemGUI(MCreator mcreator, ModElement modElement, boolean editingMode) {
 		}
 	}
 
+	private void updateGlowElements() {

```

Comment:
Nice to see you moved this into a method, but as this is only used once and only one line in the method, you can inline this.